fuzzers/005-tilegrid/ps7_int/top.py

```diff
@@ -39,9 +39,9 @@ def run():
     params = {}
 
     # NOTE: The INT_L tile has been hardcoded and it works only for the part specified in the assertion
-    assert os.getenv('XRAY_PART') == "xc7z010clg400-1"
+    assert os.getenv('XRAY_PART') == "xc7z020clg484-1"
     for isone in util.gen_fuzz_states(1):
-        params['INT_L_X0Y50'] = isone
+        params['INT_L_X18Y100'] = isone
```

Comment:
Is there a way to not hardcode this?

Reply:
Since this fuzzer eventually returns the base address of a regular interconnect tile I don't think it's possible to distinguish it from all other interconnect tiles in the tilegrid. Hence, not hardcoding would require some messing with the basicdb tilegrid and then the `add_tdb.py` or `generate_full.py` scripts.

Reply:
You misunderstand.  Is there a general relationship between the PS7 location and the interconnect choice you are making?  E.g. tile_at[PS7_tile.x + delta_x][PS7_tile.y + delta_y]?

Reply:
There is no ps7 tile from tilegrid's point of view. The ps7 tile is off-grid. It does not have a base address. Think of it as if you cut the fabric at the INT_L column and connected exposed wires to another chip which is the ps7 ARM SoC. It is only an interconnect and some pips that we know belong/are activated when the ps7 gets instantiated.